README.md

```diff
@@ -1,10 +1,11 @@
+# CFPB development guidelines
 
-> A guide for the development team at the [CFPB](http://cfpb.github.io/).
+A guide for the development team at the [CFPB](https://cfpb.github.io/).
 
 ## Standards and styles
 
-> All code in any code-base should look like a single person typed it, no matter how many people contributed.
-> - [idiomatic.js](https://github.com/rwaldron/idiomatic.js/#all-code-in-any-code-base-should-look-like-a-single-person-typed-it-no-matter-how-many-people-contributed)
+All code in any code-base should look like a single person typed it, no matter how many people contributed.
```

Comment:
Actually, I retract the 👍 . This doesn't render correctly anymore with the removal of _this_ blockquote styling, which is actually a quote.